features/step_definitions/issuer_steps.rb

```diff
@@ -34,6 +34,7 @@
 end
 
 Then('the table should have {string} in the {string} column') do |value, column_name|
+  value = Date.today.year if column_name == 'Year'
```

Comment:
thanks for fixing this! (would it make sense to also check if `"current_year"` == `value` string, just so people wouldn't put a specific year in the feature that is then just disregarded?)